bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp

```diff
@@ -91,7 +91,7 @@ void CreateLeggedOdometry(pybind11::module& module)
              py::arg("frame_index"))
         .def("change_fixed_frame",
              py::overload_cast<const std::ptrdiff_t&,
-                               const Eigen::Quaterniond&,
+                               const Eigen::Vector4d&,
                                const Eigen::Vector3d&>(&LeggedOdometry::changeFixedFrame),
```

Comment:
Since this is a problem of Python I would keep the C++ function as it is and I would modify the Python bindings by using a lambda function 
```
 .def("change_fixed_frame", [](LeggedOdometry& impl, const std::ptrdiff_t& frameIndex,
                               const Eigen::Vector4d& frameOrientation) {
             return impl.changeFixedFrame(frameIndex, frameOrientation);
```

Reply:
Addressed this in 10b810d5.